ddtrace/constants.py

```diff
@@ -44,7 +44,7 @@
 MANUAL_KEEP_KEY = "manual.keep"
 
 
-ERROR_MSG = "error.msg"  # a string representing the error message
+ERROR_MESSAGE = "error.message"  # a string representing the error message
```

Comment:
Add these imports: 
```
from ddtrace.vendor.debtcollector import removals
from ddtrace.internal.utils.deprecations import DDTraceDeprecationWarning
```

Replace line 47 with:
```suggestion
ERROR_MESSAGE = "error.message"  # a string representing the error message
@removals.remove(message="Use ERROR_MESSAGE instead." category=DDTraceDeprecationWarning)
ERROR_MSG  = ERROR_MESSAGE
```

Test the deprecation warning locally to make sure it's logged when `ERROR_MSG ` and the message is formatted correctly.

Here's how you can generate the deprecation warning:

1. Make sure ddtrace is installed locally and is an editable package. If not run the following command in a virtual environment:
 - pip install -e  [path to dd-trace-py/]

2. Generate the deprecation warning by accessing ERROR_MSG. Run:
 - python -c "from ddtrace.constants import ERROR_MSG; print(ERROR_MSG)"

Reply:
I was able to do this in a different way since you can't add a decorator in Python unless its a class or function. Instead just created another simple removal function that deprecates a constant and called it within constants.py as a normal function.

Reply:
Also I would assume that those test failures for the agent/testagent are valid and should be fixed correct? As in we are now expecting the deprecation warning when building?

Reply:
We need to fix those tests. We shouldn't use the deprecated constant internally.

Reply:
@mabdinur This is fixed and tested, resolving